INSTALL.md

```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
 languages. These drivers catch the ^ character to produce the French circumflex
 accent and do not pass it properly to GAP. For macOS users, as a workaround
 please refer to the section "GAP for macOS" below for information on
-how to install readline and section 5 on how to recompile GAP, for windows no
+how to install readline and section 5 on how to recompile GAP, for Windows no
 fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
+( https://learn.microsoft.com/en-us/windows/wsl/install ), also known as WSL.
+This can be found in the "Microsoft Store" in Windows, and allows you to install
+a full copy of Ubuntu. You can then follow the guidance for building and using
+GAP in Linux inside WSL.
 
-You might want to try different shells, using both of the two `.bat`
-files in the `bin` directory: `gap.bat` and `gapcmd.bat`. Also,
-<https://www.gap-system.org/Faq/faq.html#4> might give a remedy.
+We also support building GAP using 'Cygwin', which is a UNIX wrapper for
+Windows. Cygwin is used for making Windows release. Almost all packages are
+supported in the Windows releases.
```

Comment:
I feel "Windows release" could be misunderstood. Maybe like this
```suggestion
Windows. Cygwin is used for making the Windows installers for GAP.
Almost all packages are supported in these GAP releases for Windows.
```
Although I wonder about that final sentence, what is it meant to convey to the user? It seems to presuppose that users will assume some packages won't work, but new user have no reason to do that, to the contrary; so I think this sentence will primarily trigger the reaction "wait, some packages are not supported?! Which? And why?"

So maybe just drop that sentence, or else expand it?

Reply:
Will drop it -- older users might expect many packages to not work (as they didn't used to), but I agree ther eis no need to say that here.